package.json

```diff
@@ -44,10 +44,13 @@
   },
   "homepage": "https://github.com/middyjs/middy#readme",
   "devDependencies": {
+    "@types/aws-lambda": "^8.10.1",
+    "@types/http-errors": "^1.6.1",
+    "@types/node": "^8.0.57",
     "aws-sdk": "2.221.1",
```

Comment:
I am not a TypeScript expert, but are you sure we have to move those to dev dependencies? If we do this they won't be installed while installing middy. So in a middy app, these types libraries won't be directly available unless the user installs them explicitely. Is this a normal behaviour for TypeScript apps/libraries?

Reply:
I'm about 90% sure on this; I'll find a way to verify this - thanks for calling it out.

This weekend was busy for me so I didn't get a chance to look at making these changes just yet. It'll happen this weekend for sure (Sunday is my OSS day).